### PR TITLE
RTC-1403 - do not show app picker if OS is Windows 7 with Aero Off

### PR DESCRIPTION
**Pull request by rbarros-daitan**
If OS is windows 7 and AERO is turned off, remove "window" from sources list.

### CODE REVIEW

**Comment by rbarros-daitan** [comment]
@lneir , can you please review it?

File: symphony/Symphony/Plugins/GetWindowsPlugin.cs [hunk 1]
```diff
@@ -40,6 +41,19 @@ public void cefGetScreenMedia(JObject arguments, JavaScriptPluginCallback callba
                     values.TryGetValue("sources", out sources);
                 }
 
+                //RTC-1403 - do not show app picker if OS is Windows 7.
```
Comment (lneir) [inline symphony/Symphony/Plugins/GetWindowsPlugin.cs]: that comment is not exactly correct.  win7 and areo not enabled.

Comment (lneir) [inline symphony/Symphony/Plugins/GetWindowsPlugin.cs]: also ensure we test this on win7, win8 and win10.

Comment (rbarros-daitan) [reply symphony/Symphony/Plugins/GetWindowsPlugin.cs]: ok.

**Comment by rbarros-daitan** [comment]
Hi @lneir . Changes applied. This has been tested on Win7, Win8.1 & win10.

**Comment by lneir** [comment]
lgtm

**Pull request merged**